.gitlab-ci.yml

```diff
@@ -15,6 +15,8 @@ include:
 
 stages:
   - build
+  - publish_python_image
```

Comment:
You probably don't need separate stages for this. You can just make two jobs in the publish_artifacts stage. If the order of jobs is important, you can make one depend on the other using the `needs` keyword.

```
 needs:
    - job: .control_service_publish_job_base_image_secure
```
https://docs.gitlab.com/ee/ci/yaml/index.html#needs